Server_Hardware/Other/template_nvidia-smi_multigpu/6.0/zbx_NVidia_GPUs.yaml

```diff
@@ -0,0 +1,570 @@
+zabbix_export:
+  version: '6.4'
```

Comment:
Please change the directory according to the template version.

Reply:
Done

Reply:
Thanks for your integration